src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = (a) => {
+    if (a !== undefined) {
```

Comment:
It's good practice to explicitly check for `null` or `undefined` using `== null` instead of `!== undefined`, as it covers both `null` and `undefined` values. This would prevent any potential bugs if `null` is passed as an argument.